components/VotesList/VotesListItem.tsx

```diff
@@ -83,23 +83,22 @@ export function VotesListItem({
   const isRolled = rollCount > 0;
   const wrapperRef = useRef<HTMLTableRowElement>(null);
   const existingVote = getDecryptedVoteAsFormattedString();
-  useEffect(() => {
-    if (!options) return;
 
+  useEffect(() => {
     // if options exist but the existing decrypted vote is not one from the list,
     // then we must be using a custom input
     const decryptedVote = getDecryptedVoteAsFormattedString();
     if (decryptedVote && !findVoteInOptionsDetectEarlyVote(decryptedVote)) {
       setIsCustomInput(true);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [options, decryptedVote]);
+  }, [decryptedVote]);
 
   useEffect(() => {
     if (wrapperRef.current) {
       setWrapperWidth(wrapperRef.current.offsetWidth);
     }
-  }, [wrapperRef.current?.offsetWidth]);
+  }, [width]);
```

Comment:
this actually never worked. turns out refs never cause rerenders, so having a ref in the dependency array like this does nothing. it looked like it worked because other things would cause a rerender. using the width as the dependency solves this bug